packages/office-ui-fabric-react/src/components/Button/BaseButton.styles.ts

```diff
@@ -93,5 +91,13 @@ export const getStyles = memoizeFunction((
       margin: '0 4px',
       lineHeight: '100%'
     },
+
+    screenReaderText: {
```

Comment:
I created a mixin for this already. It's a little different, but should do the same thing.

https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/styling/src/styles/UtilityStyles.ts#L3